src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1063,6 +1064,48 @@ smart_objects::SmartObjectList MessageHelper::CreateAddCommandRequestToHMI(
   return requests;
 }
 
+smart_objects::SmartObjectSPtr MessageHelper::CreateMessageForHMI(
+    hmi_apis::messageType::eType message_type, const uint32_t correlation_id) {
+  auto message = new smart_objects::SmartObject(smart_objects::SmartType_Map);
+  auto& ref = *message;
```

Comment:
@IGapchuk suggest to use more descriptive names like `message_so` and `message_ref`

Reply:
@AKalinich-Luxoft Renamed in 36b0eadff97e5756477026c3d51f33e575f5d77a.